### PR TITLE
feat(osv): Add a purl converter for SwiftURL

### PR DESCRIPTION
**Pull request by sschuberth**
Relates to #2402.

### CODE REVIEW

File: osv/purl_helpers_test.py [hunk 1]
```diff
@@ -52,6 +52,11 @@ def test_pub(self):
     self.assertEqual('pkg:pub/characters',
                      purl_helpers.package_to_purl('Pub', 'characters'))
 
+  def test_pub(self):
+    """Test PURL generation for Swift."""
+    self.assertEqual('pkg:swift/Alamofire/Alamofire',
+                     purl_helpers.package_to_purl('SwiftURL', 'Alamofire/Alamofire'))
```
Comment (oliverchang) [inline osv/purl_helpers_test.py:58]:
```suggestion
                     purl_helpers.package_to_purl('SwiftURL', 'github.com/Alamofire/Alamofire'))
```

File: osv/purl_helpers_test.py [hunk 2]
```diff
@@ -52,6 +52,11 @@ def test_pub(self):
     self.assertEqual('pkg:pub/characters',
                      purl_helpers.package_to_purl('Pub', 'characters'))
 
+  def test_pub(self):
+    """Test PURL generation for Swift."""
+    self.assertEqual('pkg:swift/Alamofire/Alamofire',
```
Comment (oliverchang) [inline osv/purl_helpers_test.py:57]:
```suggestion
    self.assertEqual('pkg:swift/github.com/Alamofire/Alamofire',
```

**Review by oliverchang — CHANGES_REQUESTED** [review]
thanks! just a minor suggestion regarding the test

**Comment by sschuberth** [comment]
> thanks! just a minor suggestion regarding the test

Good find, I've amended my commit.

**Review by oliverchang — APPROVED** [review]

**Comment by sschuberth** [comment]
Thanks for the merge @oliverchang! Any idea by when it will be deployed in production?

**Comment by oliverchang** [comment]
This should be released this week, but the data will require a re-import. 

@hogo6002 is this something you can help with ?

**Comment by hogo6002** [comment]
> This should be released this week, but the data will require a re-import.
> 
> @hogo6002 is this something you can help with ?

Sure, I will re-import all swiftURL records

**Comment by hogo6002** [comment]
> Sure, I will re-import all swiftURL records

It turns out that `re-putting` records is enough to add `purl`. All SwiftURLs now have `purls`: https://osv.dev/vulnerability/GHSA-q36x-r5x4-h4q6
Thanks @sschuberth  for contribution!

**Pull request merged**